engine/schema/src/main/resources/META-INF/db/schema-41510to41600.sql

```diff
@@ -303,3 +303,14 @@ from
         
 -- Update name for global configuration user.vm.readonly.ui.details
 Update configuration set name='user.vm.readonly.details' where name='user.vm.readonly.ui.details';
+
+-- Update name for global configuration 'user.vm.readonly.ui.details' to 'user.vm.denied.details'
+UPDATE `cloud`.`configuration` SET name='user.vm.denied.details' WHERE name='user.vm.blacklisted.details';
```

Comment:
Config key is called - `user.vm.denylisted.details` - maybe fix one of the other one?

Reply:
thanks @rhtyd, updated to 'user.vm.denied.details'.